src/helpers/helpers.ts

```diff
@@ -0,0 +1,63 @@
+import { helpers } from '@vuelidate/validators'
+/**
+ * Tiktok Link Regex
+ * https://stackoverflow.com/questions/74077377/regular-expression-to-match-any-tiktok-video-id-and-url
+ * Cases:
+ *   - https://m.tiktok.com/h5/share/usr/6641141594707361797.html
+ *   - https://m.tiktok.com/v/6749869095467945218.html
+ *   - https://www.tiktok.com/embed/6567659045795758085
+ *   - https://www.tiktok.com/share/user/6567659045795758085
+ *   - https://www.tiktok.com/trending?shareId=6744531482393545985
+ *   - https://www.tiktok.com/@burntpizza89/video/7067695578729221378?is_copy_url=1&is_from_webapp=v1
+ *   - https://www.tiktok.com/@burntpizza89/video/is_copy_url=1&is_from_webapp=v1&item_id=7067695578729221378
+ *   - https://vm.tiktok.com/ZMF6rgvXY/
```

Comment:
It would better if we don't put real links here, put a format or template instead. Eg: 
- `https://m.tiktok.com/h5/share/usr/<id>.html`
- `https://www.tiktok.com/@<username>/video/<id>?is_copy_url=1&is_from_webapp=v1`